cves/CVE-2012-0022.yml

```diff
@@ -193,8 +204,9 @@ lessons:
     applies:
     note:
   complex_inputs:
-    applies:
-    note:
+    applies: true
+    note: This CVE is a great example of how complex/large inputs can harm
```

Comment:
I would say this is more distrust inputs than complex inputs as length does not mean complexity.

Reply:
Changed this to distrust inputs in the latest commit.